exercises/K-functions-parameters/exercise.js

```diff
@@ -1,6 +1,6 @@
 // Complete the function so that it takes input parameters
 function multiply() {
```

Comment:
inside the parenthesis you should provide parameters, which will take arguments when you call the function.
e.g. function hello(name, surname) {}. Could you re-write?

Reply:
thank you I will review it.